integrated-env

```diff
@@ -12,7 +12,8 @@
 export ADMIN_APIKEY=1111
 
 # Set this to the domain name of the site you're interested in.
-# Other interesting values depending on what you're working on: getcarina.com, support.rackspace.com.
+# Other interesting values depending on what you're working on: getcarina.com, support.rackspace.com
+# developer.rackspace.com or staging.horse (if you are not yet in production)
```

Comment:
Maybe you'll want to include this, maybe not. We're starting on new documentation and I felt bad about hammering the nexus previewer so went to setup a local environment. I spun my wheels with developer.rackspace.com not realizing that staging.horse was an allowed option as well.

Reply:
I agree that these directions could be confusing (I think the original dev was indicating that `developer.rackspace.com` is the default and the other two could be alternate values). However, I would not use staging.horse very often locally as it's just dev.rack but on a staging server. Local environment testing wouldn't spend much time simulating a staging environment, if at all. Here's my suggestion:
```bash
# Set this to the domain name of the site you're interested in.
# The current default is developer.rackspace.com. Depending on what you are working on, you might prefer support.rackspace.com here. This value should match one of the routes hosted by this instance of deconst.
export PRESENTED_URL_DOMAIN=developer.rackspace.com
```
As the getcarina site is defunct, I removed it from the example text.